src/main/java/oxgame/app/PreEndState.java

```diff
@@ -0,0 +1,61 @@
+package oxgame.app;
+
+import java.util.function.Consumer;
+
+public class PreEndState implements GameState {
+
+    private Board board;
+    private RoundBuffer playerBuffer;
+    private Player winningPlayer;
+    private Consumer<String> output;
+    private static ScoreBoard scoreBoard = new ScoreBoard();
+    private static final Integer POINTS_FOR_WIN = 3;
+    private static final Integer POINTS_FOR_LOST = 0;
+    private static final int NUMBER_OF_ROUNDS = 3;
+    private static int roundCounter = 0;
+
+    public PreEndState(Player winningPlayer, RoundBuffer playerBuffer, Consumer<String> output,Board board) {
+        this.playerBuffer = playerBuffer;
+        this.winningPlayer = winningPlayer;
+        this.output = output;
+        this.board = board;
+    }
+
+    @Override
+    public void showState() {
+        scoreBoard.addPoint(winningPlayer, POINTS_FOR_WIN);
+        playerBuffer.swapPlayers();
+        scoreBoard.addPoint(playerBuffer.takePlayer(),POINTS_FOR_LOST);
+        roundCounter++;
```

Comment:
This method cannot be static, because class implements interface GameState. 
<code>
public interface GameState {
    GameState nextState(String input) throws WrongArgumentException;
</code>